code/modules/events/grid_check.dm

```diff
@@ -0,0 +1,19 @@
+/datum/round_event_control/grid_check
+	name = "Grid Check"
+	typepath = /datum/round_event/grid_check
+	weight = 10
+	max_occurrences = 3
+
+/datum/round_event/grid_check
+	announceWhen	= 1
```

Comment:
They are actual count values. It's the delay after the event first triggers that the announcement happens.
If it's set to zero, there's no announcement though.